### ISSUE TITLE
Delete this repository  and use pxt-makerbit-mp3 instead

### ISSUE THREAD

**Issue by philipphenkel** [1010Technologies/pxt-serial-mp3#1]
I propose to delete this repository. 

**WHY?** 
We have an advanced MP3 implementation available at https://github.com/1010Technologies/pxt-makerbit-mp3. This extension is maintained and works also independently of the MakerBit board.

**Comment by HyperDuino** [comment]
ok

**Comment by philipphenkel** [comment]
@chris-torrence Please confirm if the deletion of this repository is OK from your perspective.

**Comment by chris-torrence** [comment]
That's fine. I had originally thought we should split the mp3 into a separate extension, but instead we decided to keep it combined with the makerbit.

**Comment by philipphenkel** [comment]
@chris-torrence Yeah, I remember and it turned out that your proposal was the right thing to do long-term.
